projects/ontimize-web-ngx/src/lib/components/o-form-data-component.class.ts

```diff
@@ -146,6 +162,11 @@ export class OFormDataComponent extends OBaseComponent implements IFormDataCompo
     } catch (e) {
       this.errorOptions = {};
     }
+    try {
```

Comment:
I have a doubt: what would happen if you set the select-all-on-click as true in a component and this code fails when trying to retrieve the injection token value? Would this override that 'true' value?